pennylane/transforms/qcut.py

```diff
@@ -1214,6 +1224,18 @@ def fn(x):
             )
 
     g = tape_to_graph(tape)
+
+    if auto_cutter is True or callable(auto_cutter):
+
+        cut_strategy = CutStrategy(max_free_wires=len(device_wires))
```

Comment:
Out of curiosity, is there any advantage to passing the device directly to `CutStrategy` rather than specifying `max_free_wires`?

We don't currently have the device visible with `cut_circuit`, but it wouldn't be hard to make it.